tests/integration/suite/actors/reminders/serialization.go

```diff
@@ -0,0 +1,259 @@
+/*
+Copyright 2023 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or impliei.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reminders
+
+import (
+	"bytes"
+	"context"
+	"database/sql"
+	"encoding/base64"
+	"fmt"
+	"net/http"
+	"strconv"
+	"strings"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	chi "github.com/go-chi/chi/v5"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/dapr/dapr/tests/integration/framework"
+	"github.com/dapr/dapr/tests/integration/framework/process/daprd"
+	prochttp "github.com/dapr/dapr/tests/integration/framework/process/http"
+	"github.com/dapr/dapr/tests/integration/framework/process/placement"
+	"github.com/dapr/dapr/tests/integration/framework/util"
+	"github.com/dapr/dapr/tests/integration/suite"
+)
+
+func init() {
+	suite.Register(new(serializationProtobuf))
+	suite.Register(new(serializationJSON))
+}
+
+// serializationProtobuf tests:
+// - The ability for daprd to read reminders serialized as JSON and protobuf
+// - That reminders are serialized to protobuf when the Actors API level in the cluster is >= 20
+type serializationProtobuf struct {
+	daprd   *daprd.Daprd
+	srv     *prochttp.HTTP
+	handler *serializationHTTPServer
+	place   *placement.Placement
+	db      *sql.DB
+}
+
+func (i *serializationProtobuf) Setup(t *testing.T) []framework.Option {
+	// Init placement with no maximum API level
+	i.place = placement.New(t)
+
+	// Create a SQLite database
+	sqliteOpt, db, err := daprd.WithSQLiteStateStore(t)
```

Comment:
`WithSQLightActorStateStore` should be receiving an SQLite process.

Reply:
There's no process being spawned.